### ISSUE TITLE
Plugin not working at all - French localization

### ISSUE THREAD

**Issue by Layz666** [Leo450/betterbags-gearfirst#14]
Hello !
Plugin worked fine before TWW prepatch hit, since then, it doesn't work at all.
Localization of my game is French, is that the source of the problem?

Screen captures for references.

![bag1](https://github.com/user-attachments/assets/6f285cf5-8f1f-4be1-83e7-e780c6c7d3f1)
![bag2](https://github.com/user-attachments/assets/18b75171-06d4-445e-9554-7b2bc0952ffe)
![bags3](https://github.com/user-attachments/assets/2255adc1-2ca4-4f1f-8050-af57e26bcd00)

**Comment by spn2000** [comment]
Same problem for the german client. I think something changed in the localization module in betterbags. the localized names for the section do not get set anymore
L:S("Hearthstones", "Piedras de hogar")
throws an error.

workaround for me was to modify \_retail_\Interface\AddOns\BetterBags\core\localization.lua file and add the S function before L:enable()

```
function L:S(key,localkey)
  if not self.data[key] then self.data[key] = {} end
  self.data[key][L.locale] = localkey
end
```

**Comment by Leo450** [comment]
Fixed in 1.0.11
